scapy/arch/windows/__init__.py

```diff
@@ -827,8 +827,29 @@ def read_routes():
             warning("No default IPv4 routes found. Your Windows release may no be supported and you have to enter your routes manually", onlyOnce=True)
     return routes
 
+def _get_metrics(ipv6=False):
+    """Returns a dict containing all IPv6 interfaces' metric,
```

Comment:
You mean "IPv6 (when ipv6 is True) or IPv4", or did I miss something?

Reply:
Right. Forgot to change that when copy pasting